src/pages/index.js

```diff
@@ -75,6 +71,17 @@ const IndexPage = () => (
         data-show-count="false">
     特設ページを見る</Link>
 </div>
+<div className={styles.textCenter}>
+  <h2>
+  Shop
+  </h2>
+  <p className={styles.specialDesc}>
+  SUZURIでGathering2022と10周年記念のグッズを販売中！<br />
```

Comment:
[imo]

スペースの問題かもしれないのでできればでよいのですが、
イベント名は略さずに記載しておくとよさそうと思いました 💎 
```suggestion
  SUZURIでRails Girls Gathering Japan 2022と10周年記念のグッズを販売中！<br />
```

Reply:
スペース的には問題ないのですが長くて目が泳ぐので略称として許容したいなとした意図でした！
が、絶対ではないので正式名で書きますね！